cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -1,3 +1,132 @@
+// TODO: Remove after Capital Framework v4 is merged in.
+
+.a-label {
+    display: inline-block;
+}
+
+.a-label__heading {
+    .heading-4();
+
+    display: block;
+
+    // Overwrites heading-5 margin.
+    margin-bottom: 10px;
```

Comment:
Not needed here as this is updated already in CFv4 https://github.com/cfpb/capital-framework/blob/031b458c27c28b3f9755444d866c4b8c0b1cd60b/src/cf-forms/src/atoms/label.less#L11